pkg/minikube/cluster/cluster_darwin.go

```diff
@@ -60,7 +60,6 @@ func createXhyveHost(config MachineConfig) *xhyveDriver {
 		Memory:         config.Memory,
 		CPU:            config.CPUs,
 		Boot2DockerURL: config.GetISOFileURI(),
-		BootCmd:        "loglevel=3 user=docker console=ttyS0 console=tty0 noembed nomodeset norestore waitusb=10 base host=" + constants.MachineName,
```

Comment:
@LalatenduMohanty no we don't need it with lastest xhyve drive that's why I removed it. Old version of xhyve will anyway not work with our custom iso.